src/main/mfaService.ts

```diff
@@ -0,0 +1,48 @@
+import { createHttpClient, HttpClient } from './httpClient'
+
+export type StartPhoneNumberRegistrationParams = {
+  accessToken: string
+  phoneNumber: string
+}
+
+export type VerifyPhoneNumberRegistrationParams = {
+  accessToken: string
+  phoneNumber: string
+  verificationCode: string
+}
+
+export default class MfaClient {
+  constructor(domain: string) {
+    const baseUrl = `https://${domain}/identity/v1/mfa/credentials/register/phone-number`
+    this.http = createHttpClient({
+      baseUrl
+    })
+    this.startPhoneNumberRegistrationUrl = `https://${baseUrl}/start`
+    this.verifyPhoneNumberRegistrationUrl = `https://${baseUrl}/verify`
+  }
```

Comment:
Why define a new client?

Reply:
I think for now we indeed want to keep the new methods within the existing client interface, otherwise, the client setup will change and will require updating.

Reply:
Alright I'll include it in the existing client interface.